### ISSUE TITLE
If you make a room private, any member of that room can invite others

### ISSUE THREAD

**Issue by cmaish** [JabbR/JabbR#855]
The /invitecode command is accessible to all users who are in a room (even after the owner changes the code).  What this means is that an owner can't directly control who is able to access the room.

Maybe we either need to add a flag that specifies whether the invitecode is available to all users in the room or just owners/admins, or disable the command completely for non-owners/admins.

**Comment by davidfowl** [comment]
@cmaish did you fix this one?

**Comment by cmaish** [comment]
No.  I figured that it should probably be configurable on the room somehow..
